napari/layers/utils/array_interface.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from typing import Tuple, Union
+
+import numpy as np
+
+if sys.version_info.minor < 8:
+    from typing_extensions import Protocol
+else:
+    from typing import Protocol
+
+
+class ArrayInterface(Protocol):
+    shape: Tuple[int] = (0,)
+    dtype: type = np.uint8
```

Comment:
```suggestion
    dtype: np.dtype
```

this might be too strict? but worth considering ... both dask and zarr use numpy dtypes.

Reply:
np.uint8 is not in type np.dtype

Reply:
ok, perhaps that's not the right syntax... but you understand the idea, right?

Reply:
maybe `Type[np.dtype]`

Reply:
```
type(np.uint8)
Out[4]: type
```